src/ucs/sys/sys.c

```diff
@@ -48,17 +48,23 @@
 #define UCS_PROCESS_NS_FIRST       0xF0000000U
 #define UCS_PROCESS_NS_NET_DFLT    0xF0000080U
 
+#define UCS_NS_INFO_ITEM(_id, _name, _dflt) \
+    [_id] = {.name = _name, .dflt = _dflt, \
+             .init_once = UCS_INIT_ONCE_INITIALIZER}
```

Comment:
why need both dflt and value?
let's have only value, and init it the the default

Reply:
default value is used in another function:
```
int ucs_sys_ns_is_default(ucs_sys_namespace_type_t ns)
{
    return ucs_sys_get_ns(ns) == ucs_sys_namespace_info[ns].dflt;
}
```

Reply:
```dflt``` value may be declared as ```const``` and ```value``` may be pre-initialized by default value

Reply:
ok